etcd/fs/fs_test.go

```diff
@@ -294,10 +294,6 @@ func TestFs3(t *testing.T) {
 	}
 	t.Logf("tree2: \n%s", tree2)
 
-	if _, err := memFs.Stat("/bar"); err != nil {
```

Comment:
This test is desired. `util.CopyFs(etcdFs, memFs, "/tmp/foo/bar", "/", false)` is copying `bar` to `foo` so if the test fails on this line, then CopyFs is wrong.

Reply:
yeah, gonna remove the commit where i deleted this test. my patch passes this test, anyway.